README.md

```diff
@@ -289,6 +291,20 @@ Earlier versions of the DSM Consumption Operator, including v1.1.0, v1.1.1, v1.1
 If you are upgrading from these older versions to v2.2.0, do not uninstall the existing version. Instead, we highly recommend contacting GSS for guidance and support. This will ensure a smooth upgrade process and prevent potential disruptions.
 For additional help, please refer to the support documentation or reach out to our technical support team.
 
+## Secret Store Service
+
+<img src="secret-service-logo.svg" width="250" title="Secret Store Service" id="secret-store-service">
+
+Secret Store Service is a comprehensive solution for managing secrets in vSphere, ensuring the security and integrity of the environment and providing a robust and scalable solution for securely injecting secrets into workloads.
+
+### Secret Store Service Versions 
+
+Download latest version: [Secret Store Service v9.0.0](https://packages.broadcom.com/artifactory/vsphere-distro/vsphere/iaas/secret-store/9.0.0/secret-store-service-definition.yml)
```

Comment:
We've updated this page such that it no longer includes direct links to the service manifests after the move to Broadcom's support portal. Can you update this PR to match the latest?